src/__tests__/index.spec.js

```diff
@@ -50,22 +50,44 @@ describe("index", () => {
     });
     const c = create(result.component, {
       element: thisElement,
-      layout: "layout",
+      layout: {
+        style: {
+          background: {
+            url: {
+              aStaticContentUrl: {
+                qUrl: "/media/Logo/qlik.png",
+              },
+            },
+          },
+        },
+      },
       constraints: "constraints",
     });
 
-    await c.update();
-
-    expect(thisElement.appendChild).toHaveBeenCalled();
-    expect(renderSpy).toHaveBeenCalledWith({
+    const value = {
       element: thisElement,
-      layout: "layout",
+      layout: {
+        style: {
+          background: {
+            url: {
+              aStaticContentUrl: {
+                qUrl: "/media/Logo/qlik.png",
+              },
+            },
+          },
+        },
+      },
       app: undefined,
       constraints: "constraints",
       theme: undefined,
       senseNavigation: "nav",
       multiUserAutomation: true,
       translator,
-    });
+    };
+
+    await c.update();
+
+    expect(thisElement.appendChild).toHaveBeenCalled();
+    expect(renderSpy).toHaveBeenNthCalledWith(2, value);
```

Comment:
This is a bit weird that it would render twice now I think. When looking in the nebula dev environment I still see the render count as 1, so I guess it is only in the unit test situation